tox.ini

```diff
@@ -0,0 +1,8 @@
+[tox]
+envlist = py27,py34
+[testenv]
+deps=
+    -rrequirements.txt
+    six
+    py27: mock
+commands=python -m unittest discover tests/robottelo
```

Comment:
Does Python 2's unittest test runner deal gracefully with `subText` context managers and other new features backported by unittest2?